### ISSUE TITLE
Problem in displaying all favourites

### ISSUE THREAD

**Issue by 03397** [kalkih/mini-media-player#267]
I have trouble in displaying all favourites in my list on my phone.
This list is displayed correctly on my laptop. I am missing the top two.
I cannot scroll to see them

type: 'custom:mini-media-player'
artwork: full-cover
entity: media_player.sonos_one
name: Master Bedroom
icon: 'mdi:speaker-wireless'
info: scroll
hide:
  power: true
  shuffle: false
  runtime: false
shortcuts:
  label: Favourites
  list:
    - data:
        entity_id: media_player.sonos_one
        source: Nicolas Greek
      id: media_player.select_source
      name: Nicolas Greek
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Nicolas Classics
      id: media_player.select_source
      name: Nicolas Classics
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Nicolas Laika
      id: media_player.select_source
      name: Nicolas Laika
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Nicolas Kenny G
      id: media_player.select_source
      name: Nicolas Kenny G
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Angela Classics
      id: media_player.select_source
      name: Angela Classics
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Angela Favourite
      id: media_player.select_source
      name: Angela Favourite
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Super Sport FM 104.0 (Sports Talk & News)
      id: media_player.select_source
      name: Super Sport FM 104.0 (Sports Talk & News)
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Klik FM 105.5 (Adult Hits)
      id: media_player.select_source
      name: Klik FM 105.5 (Adult Hits)
      type: service
    - data:
        entity_id: media_player.sonos_one
        source: Love fm Cyprus
      id: media_player.select_source
      name: Love fm Cyprus
      type: service

**Comment by 03397** [comment]
I have changed the layout with vertical stack cards,
Still missing from the list

type: vertical-stack
cards:
  - entity: media_player.sonos_one
    type: 'custom:mini-media-player'
    artwork: full-cover-fit
    hide:
      name: true
      power: true
      source: true
      info: true
      icon: true
      controls: true
      volume: true
      progress: true
  - entity: media_player.sonos_one
    name: Master Bedroom
    type: 'custom:mini-media-player'
    artwork: none
    info: scroll
    icon: 'mdi:speaker-wireless'
    hide:
      power: true
      shuffle: false
      runtime: false
      info: false

I am missing on line my browser in laptop and two line from my mobile,

**Comment by kalkih** [comment]
What OS and browser do you have this issue on?

**Comment by 03397** [comment]
On chrome both on my Android phone and on chrome windows 10

**Comment by 03397** [comment]
I also have tried it on edge and I am missing more lines than chrome

**Comment by kalkih** [comment]
I can't reproduce the issue, tried on desktop Chrome, Safari and on Chrome on android.
Do you have a screenshot of the issue? and is that the whole configuration? You don't nest it inside any other card than a vertical-stack?

**Comment by 03397** [comment]
I am attaching again in proper format
```
cards:
  - artwork: full-cover-fit
    entity: media_player.sonos_one
    hide:
      controls: true
      icon: true
      info: true
      name: true
      power: true
      progress: true
      source: true
      volume: true
    type: 'custom:mini-media-player'
  - entity: media_player.sonos_one
    hide:
      info: false
      power: true
      runtime: false
      shuffle: false
    info: scroll
    name: Master Bedroom
    type: 'custom:mini-media-player'
type: vertical-stack
```

If you cannot reproduce it do you think that it might be the theme you are using?
I have 13 lines and I need to to scroll? How many do you have? If less they all appear correctly.
I am attaching the list from the mini media player and then the list from the entity of home assistant.
![Screenshot_20200407-211730](https://user-images.githubusercontent.com/6150853/78705096-9fc54080-7915-11ea-8d16-a0c4412cc74f.png)
![Screenshot_20200407-211742](https://user-images.githubusercontent.com/6150853/78705100-a05dd700-7915-11ea-81b3-0c989729f69e.png)

**Comment by 03397** [comment]
As you can see that I am missing the two lines at the top. They are hidden behind the home assistant bar at the top I think.

**Comment by kalkih** [comment]
Can't you scroll in the list?

**Comment by 03397** [comment]
Only downwards.
Upwards yes but as you can see from the pictures two lines are not displayed or rather are hidden.

**Comment by kalkih** [comment]
Yes, you are right, it's hidden behind the navbar on top, not sure if I can do anything about it, but I'll look into it, thanks!

**Comment by 03397** [comment]
There was  a similar issue with the zwave devices list hidden like this and it was fixed.

So, I think it is doable.

**Comment by 03397** [comment]
@kalkih 
Did you include this in the  version 1.8.0?
Because it seems that it is no solved yet.

**Comment by kalkih** [comment]
Hello,
No, the release changelog can be found [here](https://github.com/kalkih/mini-media-player/releases/tag/v1.8.0).

**Comment by 03397** [comment]
@kalkih 
Do you have any indication when you will be able to see this?
I have uploaded a picture which displays all the favourites with no problem when there is no artist info to be displayed because of the source
![image](https://user-images.githubusercontent.com/6150853/83252999-e5371900-a1b4-11ea-9cbd-b80063eec780.png)

**Comment by kalkih** [comment]
Hello, I don't understand what you're trying to display with your comment?
The issue (if I understand correctly) is when the dropdown expands upwards and the top items are hidden behind the top bar?

**Comment by 03397** [comment]
I was trying to explain that it is with my current setup that the favorites are hidden behind the top bar.  As you can see from the picture, with the top card with nothing to display the favorites are being shown correctly.
________________________________

**Comment by kalkih** [comment]
Okay, I don't have an estimate on when it will be fixed, but will try to look at it during this weekend.
Cheers!

**Comment by 03397** [comment]
Thanks.
You work is appreciated.

________________________________

**Comment by kalkih** [comment]
I looked a bit into it and checked how the default input select ui works, it seems to always expand the dropdown downwards which eliminates this problem, I guess I could set this behaviour as well and the issue would be solved.
Having it expand dynamically like it does currently is pretty nice when there's limited screen space below the card.

**Comment by 03397** [comment]
@kalkih 
I have checked my zwave components list which is quite long and it expands upwards.
Quite weird. It used to have the exact same issue I am having now.

**Comment by kalkih** [comment]
I don't have any z-wave devices so can't check that but I've played around in the UI and most dropdowns seem to only expand downwards, then I checked the theme selection dropdown and saw that it has the same issue as this card, where the top bar is rendered above it so that the top item/items are hidden.
I'll change it to only expand downwards for now, and in the future if I find a better solution we'll change it again, or if it's fixed in core HA.

**Comment by 03397** [comment]
Ok. Let's see how it goes. Thanks.
